src/jobflow_remote/cli/backup.py

```diff
@@ -0,0 +1,134 @@
+from typing import Annotated, Optional
+
+import typer
+
+from jobflow_remote.cli.jf import app
+from jobflow_remote.cli.jfr_typer import JFRTyper
+from jobflow_remote.cli.utils import (
+    check_stopped_runner,
+    get_job_controller,
+    loading_spinner,
+    out_console,
+)
+
+app_backup = JFRTyper(
+    name="backup",
+    help="Commands for handling backup of the database",
+    no_args_is_help=True,
+)
+app.add_typer(app_backup)
+
+
+@app_backup.command()
+def create(
+    out_path: Annotated[
+        str,
+        typer.Argument(
+            help="The path of the folder where the output files will be saved",
+            metavar="PATH",
+        ),
+    ] = ".",
+    compress: Annotated[
+        bool,
+        typer.Option(
+            "--compress",
+            "-c",
+            help="Compress the output files",
+        ),
+    ] = False,
+    mongo_dir: Annotated[
```

Comment:
Maybe call it mongo-path here ? I would probably call out_dir and mongo_path the first one is the directory in which output files will be saved, the second one is the path in which mongo commands are looked for. I'm fine with anything anyway but maybe it's a bit more clear ?

Reply:
done

Reply:
Which one did you decide in the end? Seems nothing was changed. Just resolve the comment if you think it is fine to say as it is 😀

Reply:
Sorry, I had changed all the variable names except this one. It should be fine now.